nsqadmin/README.md

```diff
@@ -7,8 +7,9 @@ Read the [docs](http://nsq.io/components/nsqadmin.html)
 
 ## Working Locally
 
+ 0. install nodejs 8.x (includes npm)
  1. `$ npm install`
- 2. `$ ./gulp clean watch` or `$ ./gulp clean build`
+ 2. `$ gulp --series clean watch` or `$ ./gulp --series clean build`
```

Comment:
first `gulp` command needs the `./` prefix

Reply:
thanks, squashed in that tiny fix